src/nvim/window.c

```diff
@@ -4261,7 +4261,11 @@ int may_open_tabpage(void)
 
   cmdmod.cmod_tab = 0;         // reset it to avoid doing it twice
   postponed_split_tab = 0;
-  return win_new_tabpage(n, NULL);
+  int ok = win_new_tabpage(n, NULL);
+  if (ok == OK) {
```

Comment:
could name variable `status` so it is `status == OK` which is a bit more readable

Reply:
thx done.